scrum-of-scrums/readme.md

```diff
@@ -1,23 +1,33 @@
 # Scrum of Scrums
 
-] This content is under construction. For now, for those who are unfamiliar: [Scrum of Scrums](https://www.agilealliance.org/glossary/scrum-of-scrums)
+Scrum or scrums is a technique used to scale Scrum to a larger group. In Scrum we consider a team being too big when going over 10-12 individuals. This should be decided on a case by case basis. If the project is set up in multiple workstreams that contain a fixed group of people and a common [stand-up](../stand-ups/readme.md) meeting is slowing down productivity: scrum of scrums should be considered. You would identify the different sub teams that would almost act as separate scrum team.
 
 ## Goals
 
-What are the intended outcomes of the ceremonies?
+The goal of the scrum of scrums ceremony is to give sub teams the agility they need while not loosing visibility and coordination.

```

Comment:
An additional and main goal is to guarantee that the sub teams are achieving their sprint goals and they are in the right direction to achieve the overall project goal.